blocks/gmo-program-details/gmo-program-details.js

```diff
@@ -19,37 +23,236 @@ let viewStart, viewEnd, calendarDeliverables;
+                    span({ class: 'date-tooltip'}, 'Proposed Launch Date'),
+                    span({ class: 'campaign-date'}, 'Loading details..'),
+                ),
+                span({ class: 'driver-text'}, 'Project Owner: Loading details..'),
+                div(
+                    { class: 'header-row3 data-element'},
+                    span({ class: 'icon icon-release-tier'}),
+                    span({ class: 'release-tier'}, 'Release Tier: Loading details..'),
+                ),
+                div(
+                    { class: 'header-row3 data-element'},
+                    span({ class: 'icon icon-productGroup'}),
+                    span({ class: 'productGroup'}, 'Loading details..'),
+                ),
+             ),
+        ),
+    );
+
+    // tab wrapper
+    const tabWrapper = div(
+        { class: 'tab-wrapper'},
+        div({ id: 'tab1toggle', class: 'tabBtn active', 'data-target': 'tab1'}, 'Overview'),
+        div({ id: 'tab2toggle', class: 'tabBtn', 'data-target': 'tab2'}, 'Deliverables'),
+        div({ id: 'tab3toggle', class: 'tabBtn', 'data-target': 'tab3'}, 'Calendar'),
+    );
+
+    // overview tab
+    const overviewTab = div(
+        { id: 'tab1', class: 'two-column overview tab'},
+        div(
+            { class: 'overview-wrapper'},
+            span({ class: 'h1 overview-heading'}, 'At a Glance'),
+            div(
+                { class: 'product-overview-wrapper'},
+                span({ class: 'h3'}, 'Marketing Goal'),
+                div({ class: 'overview paragraph hide-overflow'}, ' '),
+                div({ class: 'button no-bg read-more'}, 'Read more'),
+            ),
+            div(
+                { class: 'product-value-wrapper'},
+                span({ class: 'h3'}, 'Product Value'),
+                div({ class: 'description paragraph hide-overflow'}, ' '),
+                div({ class: 'button no-bg read-more'}, 'Read more'),
+            ),
+            div(
+                { class: 'kpis-wrapper'},
+                span({ class: 'h3'}, 'KPIs to Measure Success'),
+            ),
+            div(
+                { class: 'kpis-wrapper market-wrapper'},
+                span({ class: 'h3'}, 'Target Market Area'),
+            ),
+            div(
+                { class: 'use-cases-wrapper inactive'},
+                span({ class: 'h3'}, 'Hero Use Cases'),
+                div(
+                    { class: 'tags-wrapper'},
+                    div({ class: 'use-case-tag'}, 'Text to Image'),
+                    div({ class: 'use-case-tag'}, 'Use Case 2'),
+                )
+
+            ),
+            div(
+                { class: 'main-message-wrapper inactive'},
+                span({ class: 'h3'}, 'Main Message'),
+                span(
+                    { class: 'description'},
+                    'A major genAI release of the Photoshop beta app that delivers new and enhanced generative AI capabilities.',
```

Comment:
@mdickson-adbe is this description for Marketing Goal?

Reply:
This was a field that we never actually did anything with.. I can actually just delete it because it's never shown (note the 'inactive' class)